### PR TITLE
Link to GitHub contributors in about dialog

### PR DESCRIPTION
**Pull request by stefan-kolb**
# Remove AUTHORS file

## Reason
  
- AUTHORS file is hard to maintain
- Github/Git is single source of truth
- some authors did not want to be listed by name -> Github profile is maintained by them or NOT if they do not want to
- AUTHORS list also includes 12345 and duplicate names that look odd (and that was the reason why we had mailmap and such to avoid it)
  
## Solutions

- link to https://github.com/JabRef/jabref/graphs/contributors
- autogenerate file entirely as AUTHORS and ignore duplicates etc
- use something like https://github.com/mgechev/github-contributors-list to generate file
  
## Decision

- Change info dialog to link to contributors
- global RM search for AUTHORS (.gitattributes)
- RM .mailmap
- disable fullscreen and resize

image

- [ ] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by koppor** [comment]
Alternatives: Maintain "All Contributors" (contributing ideas, discussions, feedback): https://github.com/all-contributors/all-contributors. This is much effort and causes huge mental workload. We want to spend our energy on fixing issues and providing issue feedback.

We accept that contributors contributing something before the GitHub area (on SourceForge) are not listed explicitly after merging this PR. They are still listed in v5.1 and earlier. They can also be foudn at the link. https://github.com/JabRef/jabref/blob/3ebba86978b6f5a714b7dec988a2aed04780d84a/scripts/generate-authors.sh#L54

**Comment by koppor** [comment]
While working on it, we also renamed "Developers" to "Maintainers", since @JabRef/developers are more the overall maintainers. IMHO, this is more welcoming to contributors, because they are also developers, but not maintainers of JabRef. We as maintainers are also developers.

**Comment by Siedlerchr** [comment]
What about core developers and contributer?

**Comment by tobiasdiez** [comment]
+ 1 for removing the authors and also for this
> use something like https://github.com/mgechev/github-contributors-list to generate file

We could put the file at `jabref.org/contributors` and link it in the about dialog (and from the main website).

**Comment by koppor** [comment]
>> use something like [mgechev/github-contributors-list](https://github.com/mgechev/github-contributors-list) to generate file 

> We could put the file at `jabref.org/contributors` and link it in the about dialog (and from the main website).

Another tool is "true-contributors", which also lists issue reporters, etc. --> https://github.com/100Automations/true-github-contributors#readme. Not too bad, too, IMHO.

**Comment by koppor** [comment]
Using `jabref.org/contributors` instead of https://github.com/JabRef/jabref/graphs/contributors will be part of a follow-up PR as I don't currently have the time to implement that.

**Review by Siedlerchr — APPROVED** [review]

File: src/main/java/org/jabref/gui/help/AboutDialogViewModel.java
Comment (tobiasdiez) [inline src/main/java/org/jabref/gui/help/AboutDialogViewModel.java:38]: Since it's a static string, there is no need for an observable property. You can simply set it in the fxml via `text = %JabRef...`.

**Review by tobiasdiez — CHANGES_REQUESTED** [review]
LGTM, one small comment, then it can be merged.

**Review by calixtus — APPROVED** [review]
I don't really have a strong opinion on these changes.
As long as somehow the contributors who like to be known (and also those of the sourceforge-era) get credit for their work somehow, I'm fine with the changes.

**Comment by koppor** [comment]
> As long as somehow the contributors who like to be known (and also those of the sourceforge-era) get credit for their work somehow, I'm fine with the changes.

I did not receive any explicit request. Thus, I would continue here and handle "exceptions" as soon as someone raises his voice.

**Review by koppor — APPROVED** [review]

**Pull request merged**
